tasks/coverage/src/runtime/getBabelHelpers.js

```diff
@@ -1,122 +1,138 @@
+  babel.decorate = require('../../babel/packages/babel-runtime/helpers/decorate');
+  babel.defaults = require('../../babel/packages/babel-runtime/helpers/defaults');
+  babel.defineAccessor = require('../../babel/packages/babel-runtime/helpers/defineAccessor');
+  babel.defineEnumerableProperties = require('../../babel/packages/babel-runtime/helpers/defineEnumerableProperties');
+  babel.defineProperty = require('../../babel/packages/babel-runtime/helpers/defineProperty');
+  babel.dispose = require('../../babel/packages/babel-runtime/helpers/dispose');
+  babel.extends = require('../../babel/packages/babel-runtime/helpers/extends');
+  babel.get = require('../../babel/packages/babel-runtime/helpers/get');
+  babel.getPrototypeOf = require('../../babel/packages/babel-runtime/helpers/getPrototypeOf');
+  babel.identity = require('../../babel/packages/babel-runtime/helpers/identity');
+  babel.importDeferProxy = require('../../babel/packages/babel-runtime/helpers/importDeferProxy');
+  babel.inherits = require('../../babel/packages/babel-runtime/helpers/inherits');
+  babel.inheritsLoose = require('../../babel/packages/babel-runtime/helpers/inheritsLoose');
+  babel.initializerDefineProperty = require('../../babel/packages/babel-runtime/helpers/initializerDefineProperty');
+  babel.initializerWarningHelper = require('../../babel/packages/babel-runtime/helpers/initializerWarningHelper');
+  babel.instanceof = require('../../babel/packages/babel-runtime/helpers/instanceof');
+  babel.interopRequireDefault = require('../../babel/packages/babel-runtime/helpers/interopRequireDefault');
+  babel.interopRequireWildcard = require('../../babel/packages/babel-runtime/helpers/interopRequireWildcard');
+  babel.isNativeFunction = require('../../babel/packages/babel-runtime/helpers/isNativeFunction');
+  babel.isNativeReflectConstruct = require('../../babel/packages/babel-runtime/helpers/isNativeReflectConstruct');
+  babel.iterableToArray = require('../../babel/packages/babel-runtime/helpers/iterableToArray');
+  babel.iterableToArrayLimit = require('../../babel/packages/babel-runtime/helpers/iterableToArrayLimit');
+  babel.jsx = require('../../babel/packages/babel-runtime/helpers/jsx');
+  babel.maybeArrayLike = require('../../babel/packages/babel-runtime/helpers/maybeArrayLike');
+  babel.newArrowCheck = require('../../babel/packages/babel-runtime/helpers/newArrowCheck');
+  babel.nonIterableRest = require('../../babel/packages/babel-runtime/helpers/nonIterableRest');
+  babel.nonIterableSpread = require('../../babel/packages/babel-runtime/helpers/nonIterableSpread');
+  babel.nullishReceiverError = require('../../babel/packages/babel-runtime/helpers/nullishReceiverError');
+  babel.objectDestructuringEmpty = require('../../babel/packages/babel-runtime/helpers/objectDestructuringEmpty');
+  babel.objectSpread = require('../../babel/packages/babel-runtime/helpers/objectSpread');
+  babel.objectSpread2 = (obj, rest) => ({ ...obj, ...rest });
```

Comment:
We need to use t he helpers, that's how things are run in runtimes.

Reply:
Ok, Let me figure out how the difference between `objectSpread` and `objectSpread2`, and then report it to `Babel`